app/components/AgendaOverview.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { StakePoolStyles } from './views/ViewStyles';
+import AgendaClose from './AgendaClose';
+import KeyBlueButton from './KeyBlueButton';
+
+class AgendaOverview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      choice: this.props.selectedChoice,
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (this.props.selectedChoice != nextProps.selectedChoice) {
+      this.setState({choice: nextProps.selectedChoice});
+    }
+  }
+  render() {
+    return (
+          <div style={StakePoolStyles.agenda}>
+            <div style={StakePoolStyles.agendaOverview}>
+              <div style={StakePoolStyles.agendaOverviewTitleArea}>
+                <AgendaClose onClick={() => this.props.closeCurrentAgenda()}/>
+                <div style={StakePoolStyles.agendaOverviewTitleName}>{this.props.agenda.getId()}</div>
+              </div>
+              <div style={StakePoolStyles.agendaOverviewMiddle}>
+                <div style={StakePoolStyles.agendaOverviewText}>
+                  <span>{this.props.agenda.getDescription()}</span><br/>
+                  <span style={StakePoolStyles.agendaOverviewAgendaId}>
+                    Agenda ID: <span style={StakePoolStyles.agendaOverviewAgendaIdId}>#{this.props.agenda.getId()}</span>
+                  </span>
+                  <br/>
+                  <br/>
+                  <span>
+                    Once the majority of the PoW miners have upgraded (75% of the 100 most recent blocks are at the latest version) and the majority of the PoS
+                    miners have upgraded (75% of the votes in a 2016 block interval) have upgraded, the voting process begins.
+                  </span><br/>
+                  <a target="_blank" href="http://decred.org" style={StakePoolStyles.agendaOverViewReadMore}>Read more »</a>
```

Comment:
I've removed this link for now, untli we can get a better idea where this should go,
docs.decred.org?  DCP on github?